.gitignore

```diff
@@ -0,0 +1,2 @@
+aternos-bot-venv
```

Comment:
in general, .venv is folder name of virtual env. 
Files / folder starts with . is hidden in *nix. - rename folder with .venv and add to .gitignore